code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -97,7 +98,19 @@
 
 /obj/item/ammo_storage/magazine/c45/empty
 	starting_ammo = 0
-
+/*
+/obj/item/ammo_storage/magazine/c45/rubber
+	name = "magazine (.45 rubber)"
+	desc = "A magazine designed to load rubber bullets into common .45 pistols. Holds 8 rounds."
+	icon_state = "45R"
+	ammo_type = "/obj/item/ammo_casing/c45r"
+
+/obj/item/ammo_storage/magazine/c45/practice
+	name = "magazine (.45 practice)"
+	desc = "A magazine designed to load practice bullets into common .45 pistols. Holds 8 rounds."
+	icon_state = "45P"
+	ammo_type = "/obj/item/ammo_casing/c45p"
+*/ //Gotta port the Polaris caliber system first else there's no point.
```

Comment:
With the way our code worked guns, magazines, boxes and bullets were very strict in what they could load (into).